pkg/controllers/dynakube/otelc/statefulset/env.go

```diff
@@ -107,3 +120,33 @@ func getEnvs(dk *dynakube.DynaKube) []corev1.EnvVar {
 
 	return envs
 }
+
+func getProxyEnvValue(envVar string, src *value.Source) corev1.EnvVar {
```

Comment:
the function a bit confusing, it;s concrete implementation for `getDynakubeProxyEnvValue`

Reply:
Renamed [7bc2317](https://github.com/Dynatrace/dynatrace-operator/pull/4630/commits/7bc23172d110807e4fb55fd5a450a88de1bae893)